### PR TITLE
#169230797 Search form and Popular Cities

### PR DESCRIPTION
**Pull request by WillyBoris**
#### What does this PR do?
 Make Search form 
#### Description of Task to be completed?
- Add search fields and search button
- Add popular cities that have been recently searched
#### How should this be manually tested?
- Clone this repo
- cd . into ft-search-form-169230797  branch
- install the dependencies (npm install)
- Start the app
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#169230797](https://www.pivotaltracker.com/story/show/169230797)
#### Screenshot?
image

### CODE REVIEW

**Comment by kipropbrian** [comment]
There is nothing here.

**Comment by WillyBoris** [comment]
> There is nothing here.

> I corrected it, you can see the changes

File: src/App.js [hunk 1]
```diff
@@ -1,8 +1,96 @@
...
+dotenv.config();
+
+class App extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      cord: "",
+      city: "",
+      data: [],
+      recentCities:['kigali', 'dodoma', 'nairobi'],
```
Comment (victorkarangwa) [inline src/App.js]: Let's just remove these hardcoded data

File: src/components/Form.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+            type="text"
+            placeholder="Type city name"
+            value={city}
+            onChange={handleChange}
+          />
+          <button onClick={handleSubmit}><img src={img} alt="searchbtn" /></button>
+        </form>
+        <div className="popular-searches">
+          {
+            this.props.state.recentCities.map((item, index) => {
```
Comment (NiyoEric) [inline src/components/Form.js]: can use es6 to destructure *recentCities* from the state?

**Review by victorkarangwa — COMMENTED** [review]
Get rid of hardcoded data

**Review by NiyoEric — CHANGES_REQUESTED** [review]

**Review by ivymwende — COMMENTED** [review]
Nice Work! However please add font styling to the form to make it more visually appealing.

**Review by NiyoEric — APPROVED** [review]

**Comment by WillyBoris** [comment]
> Nice Work! However please add font styling to the form to make it more visually appealing.

>I have added the font you asked for

**Review by victorkarangwa — APPROVED** [review]

**Review by hezzie — APPROVED** [review]

**Review by victorkarangwa — APPROVED** [review]

**Pull request merged**